### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by melneubert**
Early dashboard. Has steps, distance, and battery life.

To do:
-Possibly cache fitbit API responses
-Handle WellCat API features
-Toggle between days

### CODE REVIEW

**Review by ipavl — APPROVED** [review]

**Comment by ipavl** [comment]
Thoughts on something like "0 steps [so far] today" to give more context than just "0 steps", or some other indicator of what the numbers represent?

File: App/Containers/Dashboard.js [hunk 1]
```diff
@@ -1,17 +1,78 @@
...
+        deviceBattery: deviceInfo[0].battery
+      })
+    }
+  }
+
+  async fetchDayOfSummaryData () {
+    let date = Moment().format('YYYY[-]MM[-]DD')
+    let accessToken = await AsyncStorage.getItem(StorageKeys.FITBIT_ACCESS_TOKEN)
+
+    fetch(`https://api.fitbit.com/1/user/-/activities/date/${date}.json`, {
```
Comment (ipavl) [inline App/Containers/Dashboard.js:41]: If you're just doing for the current day, I think `today.json` may also work, but maybe once we abstract API stuff it won't matter anyways.

File: App/Containers/Dashboard.js [hunk 2]
```diff
@@ -1,17 +1,78 @@
...
+
+  getBatteryIcon () {
+    if (this.state.deviceBattery === 'High') {
+      return 'battery-full'
+    } else if (this.state.deviceBattery === 'Medium') {
+      return 'battery-three-quarters'
+    } else if (this.state.deviceBattery === 'Low') {
+      return 'battery-quarter'
+    } else {
+      return 'battery-empty'
```
Comment (mmacke01) [inline App/Containers/Dashboard.js]: hard-coded strings make me :(

**Comment by mmacke01** [comment]
👍

**Comment by Andrewwelton** [comment]
👍

**Pull request merged**